src/components/Media/Media.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./Media.css";
+import Img from "gatsby-image"
+import {useStaticQuery, graphql} from "gatsby"
```

Comment:
Both the `Img` and `{useStaticQuery, graphql}` are imports which aren't being used in the page. Kindly remove them. Also would recommend you to use a code editor like VS Code or Sublime Text and setup a linting plugin, that will highlight unused variables and imports in the file, makes the code cleaner and much easier to read. I use ESLint on VS Code.

Reply:
Thanks for the tip. Pushing the changes.